.github/workflows/release_update_docs_px_dev.yaml

```diff
@@ -0,0 +1,102 @@
+---
+name: update-docs-px-dev-on-release
+on:
+  push:
```

Comment:
we should do this on release published instead of on push to the tag. See: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
I wasn't sure how to scope the release publish event to just the vizier releases, which is why I went for that. My latest change should hopefully test that the `release/vizier/*` git ref must match while using the publish event.